src/radical/pilot/agent/rm/lsf_summit.py

```diff
@@ -172,83 +174,77 @@ def __init__(self, cfg, session):
     def _configure(self):
 
         lsf_hostfile = os.environ.get('LSB_DJOB_HOSTFILE')
-        if lsf_hostfile is None:
-            msg = "$LSB_DJOB_HOSTFILE not set!"
-            self._log.error(msg)
-            raise RuntimeError(msg)
-
-        lsb_mcpu_hosts = os.environ.get('LSB_MCPU_HOSTS')
-        if lsb_mcpu_hosts is None:
-            msg = "$LSB_MCPU_HOSTS not set!"
-            self._log.error(msg)
-            raise RuntimeError(msg)
-
-        # parse LSF hostfile
-        # format:
-        # <hostnameX>
-        # <hostnameX>
-        # <hostnameY>
-        # <hostnameY>
+        if not lsf_hostfile:
+            raise RuntimeError("$LSB_DJOB_HOSTFILE not set!")
```

Comment:
I think this writes the error/exception to stderr whereas previously we wrote to the
log file. Is the change intended?  Should we stick to one over the other in all
components/subcomponents?

Reply:
It is caught in the agent when the RP is constructed,  and will be logged there, including the exception trace.  Exception should be used as always, and be logged where they are caught (and they should *always* we caught - if not, we need to fix that),  Error logs near the raise make sense if it gives additional information - but not so much if we end up with the same message twice.

Reply:
Why is the check for `LSB_MCPU_HOSTS` removed?